common/test/acceptance/tests/studio/test_studio_course_team.py

```diff
@@ -26,6 +26,21 @@ def _make_user(self, username):
         ).visit()
         return user
 
+    def _update_user(self, **update_values):
+        """
+        Update other_user with provided values
+
+        Arguments:
+            `update_values`: dictionary containing values of attributes to be updated
+        """
+        AutoAuthPage(
+            self.browser, no_login=True, username=self.other_user.get('username'),
+            email=self.other_user.get('email'), password=self.other_user.get('password'),
```

Comment:
This is intentionally done this way as only requirement here is to update `other_user` which is already made to be used in tests. Providing all data of `self.other_user` to `user_info` will just be a repetition.
If you think there might be other cases where we need `user_info` but not `self.other_user` please let me know.